doc/influxql-2.md

```diff
@@ -0,0 +1,910 @@
+# InfluxQL & Kapacitor 2.0
+This document puts forth some ideas for the new data model of InfluxDB and a query language that unites Influx and Kapacitor into one. The goal is to have a query language that is easy to use and fits with the model of working with time series data. It should be extensible and easy for users to add new functions and build on top of some key primitives.
+
+The rollout of this should be as an additional endpoint in InfluxDB and Kapacitor that can work with the existing data model or work with the new data model.
+
+## Proposed Data Model & Terms
+
+The new data model's goal is to simplify things for the users, while keeping the ability to represent other data models like InfluxDB 1.x, OpenTSDB, and Prometheus. The primary goal is to remove questions from the user about how to organize things and also let the data structures improve the clarity of the query language. The new model consists of databases, which have a retention period (how long they keep data around) and time series are identified by tag key/value pairs (note that measurements and fields are no longer a concept that exists). A series is a time ordered collection of time/value pairs. Values in a series must all be of the same type, either bool, int64, uint64, float64, or string. Queries can span multiple databases and can write into multiple databases so removing the separate retention policy heirarchy won't limit functionality.
+
+### Definitions
+
+The following definitions encompass the entire vocabulary of the data model. Where appropriate, JSON examples are given, but the protocol will also support a line or protobuf based structure.
```

Comment:
I think we'll be working out the result format in the next four weeks or so. Will add more detail when we have it.